apps/src/blockly/addons/cdoWorkspaceSvg.js

```diff
@@ -27,13 +27,6 @@ export default class WorkspaceSvg extends GoogleBlockly.WorkspaceSvg {
     this.trashcan = new Blockly.Trashcan(this);
     var svgTrashcan = this.trashcan.createDom();
     this.flyout_.svgGroup_.appendChild(svgTrashcan);
-    this.pluginManager_.addPlugin({
```

Comment:
checking my understanding -- we don't need to add the trashcan plugin anymore because we're registering CdoBlockDragger, which now controls the trashcan (basically, disabling and opening/closing the lid)?

Reply:
A lot of things have changed in Blockly internal logic (this is a major version bump), so I'm not 100% sure on the details here, but I think the key things are:
- The trashcan now adds itself to the ComponentManager (PluginManager was renamed to ComponentManager) in its own init method: https://github.com/google/blockly/blob/master/core/trashcan.js#L375. Since we subclass the trashcan, this was the change that made it so that we don't have to call `addPlugin` here
- Registering the `CdoBlockDragger` is necessary now because `Gesture` instantiates a `BlockDragger` from the registry: https://github.com/google/blockly/blob/master/core/gesture.js#L442.

Reply:
ahhhh, okay, that makes more sense. thank you!